content/docs/reference/cli-install.md

```diff
@@ -7,41 +7,71 @@ updatedOn: '2023-11-08T18:42:10.264Z'
 
 This section describes how to install the Neon CLI and connect via web authentication or API key.
 
-### Prerequisites
+<Tabs labels={["npm", "macOS", "Windows", "Linux"]}>
 
-Before installing, ensure that you have met the following prerequisites:
+<TabItem>
 
-- Node.js 18.0 or higher. To check if you already have Node.js, run the following command:
+To install the Neon CLI via [npm](https://www.npmjs.com/package/neonctl):
 
-    ```shell
-    node -v
-    ```
+```shell
+npm i -g neonctl
+```
 
-- The `npm` package manager.  To check if you already have `npm`, run the following command:
+Requires [Node.js 18.0](https://nodejs.org/en/download/) or higher.
 
-   ```shell
-   npm -v
-   ```
 
-  If you need to install `Node.js` or `npm`, refer to instructions on the [official nodejs page](https://nodejs.org) or use the [Node version manager](https://github.com/nvm-sh/nvm).
+</TabItem>
 
-### Install
+<TabItem>
 
-To install the Neon CLI, run the following command:
+To install the Neon CLI via [Homebrew](https://formulae.brew.sh/formula/neonctl):
 
-```shell
-npm i -g neonctl
+```bash
+brew install neonctl
 ```
 
-Homebrew is also supported:
+</TabItem>
+
+<TabItem>
+
+Download the [neonctl-win.exe binary](https://github.com/neondatabase/neonctl/releases/tag/v1.24.0).
```

Comment:
I think it would be better to move word binary outside of link:

<img width="458" alt="image" src="https://github.com/neondatabase/website/assets/2598671/cc09958d-123a-4d4a-a0c9-e9b16db7e433">

Link on Linux tab:

<img width="425" alt="image" src="https://github.com/neondatabase/website/assets/2598671/499d893a-9b6b-4da8-9ed8-3f593c846906">